ts/src/bitmex.ts

```diff
@@ -289,95 +290,96 @@ export default class bitmex extends Exchange {
         //        "optionStrikeRound": null,
         //        "optionStrikePrice": null,
         //        "optionMultiplier": null,
-        //        "positionCurrency": "LTC",
-        //        "underlying": "LTC",
+        //        "positionCurrency": "XBT", // can be empty for spot markets
+        //        "underlying": "XBT",
         //        "quoteCurrency": "USDT",
-        //        "underlyingSymbol": "LTCT=",
+        //        "underlyingSymbol": "XBTT=", // can be empty for spot markets
         //        "reference": "BMEX",
-        //        "referenceSymbol": ".BLTCT",
+        //        "referenceSymbol": ".BXBTT", // can be empty for spot markets
         //        "calcInterval": null,
         //        "publishInterval": null,
         //        "publishTime": null,
-        //        "maxOrderQty": 1000000000,
-        //        "maxPrice": 1000000,
-        //        "lotSize": 1000,
-        //        "tickSize": 0.01,
-        //        "multiplier": 100,
-        //        "settlCurrency": "USDt",
-        //        "underlyingToPositionMultiplier": 10000,
+        //        "maxOrderQty": "1000000000",
+        //        "maxPrice": "1000000",
+        //        "lotSize": "1000",
+        //        "tickSize": "0.5",
+        //        "multiplier": "1",
+        //        "settlCurrency": "USDt", // can be empty for spot markets
+        //        "underlyingToPositionMultiplier": "1000000",
         //        "underlyingToSettleMultiplier": null,
-        //        "quoteToSettleMultiplier": 1000000,
+        //        "quoteToSettleMultiplier": "1000000",
         //        "isQuanto": false,
         //        "isInverse": false,
-        //        "initMargin": 0.03,
-        //        "maintMargin": 0.015,
-        //        "riskLimit": 1000000000000,
-        //        "riskStep": 1000000000000,
+        //        "initMargin": "0.01",
+        //        "maintMargin": "0.005",
+        //        "riskLimit": "1000000000000", // can be null for spot markets
+        //        "riskStep": "1000000000000", // can be null for spot markets
         //        "limit": null,
         //        "capped": false,
         //        "taxed": true,
         //        "deleverage": true,
-        //        "makerFee": -0.0001,
-        //        "takerFee": 0.0005,
-        //        "settlementFee": 0,
-        //        "insuranceFee": 0,
-        //        "fundingBaseSymbol": ".LTCBON8H",
-        //        "fundingQuoteSymbol": ".USDTBON8H",
-        //        "fundingPremiumSymbol": ".LTCUSDTPI8H",
-        //        "fundingTimestamp": "2022-01-14T20:00:00.000Z",
+        //        "makerFee": "-0.0001",
+        //        "takerFee": "0.00075",
```

Comment:
@ttodua I don't thing the API returns stringified numbers 🤔

Reply:
hm let me recheck what went wrong on my side.

Reply:
dang, I posted stringified comments yeah, my bad.

Reply:
seems there were no more comments needed (as previous stringified comments were mistake) so, only these few things were left